### PR TITLE
v1.14.0 - Add new clock icon

### PR DESCRIPTION
**Pull request by meselgroth**

### CODE REVIEW

File: src/img/icons/tools/clockGreen.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
```
Comment (ashleynolan) [inline src/img/icons/tools/clockGreen.svg]: The idea of using You should be able to change the colour of SVGs through the `fill` CSS property.  We try and keep the icons in this repo as the default colour they need to be, and then change the colour using styling (as this is better for performance than loading in a new file for each colour).

File: src/img/icons/tools/clock.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
```
Comment (ashleynolan) [inline src/img/icons/tools/clock.svg]: Where is this clock going to be used?

Main reason to ask is we have a couple of clock icons in the repo already that are in use on Search and Payments:

- [Clock](https://github.com/justeat/f-icons/blob/master/src/img/icons/general/clock.svg)
- [clock-filled](https://github.com/justeat/f-icons/blob/master/src/img/icons/general/clock--filled.svg)

Not sure if the designer working on this knows these are already there, but would be good not to have lots of very similar icons in use unless needed.

Comment (meselgroth) [reply src/img/icons/tools/clock.svg]: Will check...

Comment (meselgroth) [reply src/img/icons/tools/clock.svg]: Pieter-Jan Voskamp:
They're 2 different icons in our set. The clock.svg is for times, the one included on Zeplin for this experiment is for ETA so if possible please use the provided icon. 
the clock--filled.svg is the old icon set
and the clock.svg is used for "Pre-order" so having separate icons for these data points would be best.

Comment (meselgroth) [reply src/img/icons/tools/clock.svg]: Have renamed it to stopwatch and moved it in general so it is in the same folder

File: src/img/icons/general/stopwatch.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
```
Comment (DamianMullins) [inline src/img/icons/general/stopwatch.svg]: Can you run the SVG through [SVGOMG](https://jakearchibald.github.io/svgomg/) and tick the setting to remove the width and height attributes, please?

Also, you might find it useful to add a matching CSS class where you can set the width and height e.g. [the other icons in the `general/` directory have matching classes](https://github.com/justeat/f-icons/blob/master/src/scss/partials/_general.scss).

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,7 +3,7 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-v1.13.0
+v1.14.0
```
Comment (DamianMullins) [inline CHANGELOG.md]: You need to add a new item to the changelog for 1.14.0

Comment (ashleynolan) [reply CHANGELOG.md]: Yep – if you do a rebase on master too – I've just merged in `v1.13.1` too, so if you can add this entry above that in the CHANGELOG that'd be 👍

File: src/img/icons/tools/clock.svg (hunk 2 above)
Comment (ashleynolan) [reply src/img/icons/tools/clock.svg]: Cheers for confirming – makes sense now.  I'll look to get the old icon (clock--filled.svg) removed as should only be global icons in this repo.

**Pull request merged**